### PR TITLE
RamSoft dicomProxy Updated version: 0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1

### PR DESCRIPTION
**Pull request by ghost**
RamSoft dicomProxy updated the version 0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1, RamSoft DicomProxy from release pipeline

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17663)

### CODE REVIEW

**Comment by ramsoft-inc** [comment]
@ItzLevvie  can you please help me to review my pull request?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Seeing a hash mismatch:
image

**Comment by ramsoft-inc** [comment]
@ItzLevvie  seems something wrong I check the hash after downloading the exe in local, it's same hash
`winget hash -f DICOM-PROXY-0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1.exe`

image


 https://github.com/microsoft/winget-cli/issues/714

File: manifests/r/RamSoft/DicomProxy/0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1/RamSoft.DicomProxy.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+PackageLocale: "en-US"
+Publisher: "RamSoft Inc."
+PackageName: "RamSoft DicomProxy"
+License: "Copyright © 1994-2021 RamSoft Inc. All rights reserved."
+ShortDescription: "DicomProxy is a windows appplication running as a windows service to assists local modalities to transfer dicom studies to a remote PACS server."
+Installers: 
+ - Architecture: "x64"
+   InstallerType: "exe"
+   InstallerUrl: "https://software.ramsoft.com/dicomproxy/DICOM-PROXY-0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1.EXE"
+   InstallerSha256: "2FEA46FAE58A8FF1F45B143440FD832D5A00F182ED832BE16D097F457F4350E3"
```
Comment (ItzLevvie) [inline manifests/r/RamSoft/DicomProxy/0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1/RamSoft.DicomProxy.yaml:12]:
```suggestion
   InstallerSha256: "32D9F632C111BDFD7702528C90C3EE8377A8FEE8C1CB65A6508FB2F6E21154C9"
```
I did another try today:
image

This is the hash I'm seeing when downloading it locally:
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/r/RamSoft/DicomProxy/0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1/RamSoft.DicomProxy.yaml (hunk 1 above)
Comment (ramsoft-inc) [reply manifests/r/RamSoft/DicomProxy/0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1/RamSoft.DicomProxy.yaml]: @ItzLevvie  How do you get this sha256?

**Comment by ramsoft-inc** [comment]
@ItzLevvie  With new hash256 pipeline is failed and suggest to put the old SHA256 which was before.

**Comment by denelon** [comment]
It looks like the URL is redirecting.

```YAML
wingetcreate update RamSoft.DicomProxy https://software.ramsoft.com/dicomproxy/DICOM-PROXY-0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1.EXE
Retrieving latest manifest for RamSoft.DicomProxy
Downloading and parsing: https://software.ramsoft.com/dicomproxy/DICOM-PROXY-0.2.182-ci.2.EXE...

Generating a preview of your manifests...
Version manifest preview:
# Created using wingetcreate 0.2.0.29
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json

PackageIdentifier: RamSoft.DicomProxy
PackageVersion: 0.2.182-ci.2
DefaultLocale: en-US
ManifestType: version
ManifestVersion: 1.0.0


Installer manifest preview:
# Created using wingetcreate 0.2.0.29
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json

PackageIdentifier: RamSoft.DicomProxy
PackageVersion: 0.2.182-ci.2
Installers:
- Architecture: x64
  InstallerType: exe
  InstallerUrl: https://software.ramsoft.com/dicomproxy/DICOM-PROXY-0.2.182-ci.2.EXE
  InstallerSha256: 0412C79C0420C9110368B4B3C1F96497D9D6A2B8A2FF300239652B36D1D5DE6E
ManifestType: installer
ManifestVersion: 1.0.0


Default locale manifest preview:
# Created using wingetcreate 0.2.0.29
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json

PackageIdentifier: RamSoft.DicomProxy
PackageVersion: 0.2.182-ci.2
PackageLocale: en-US
Publisher: RamSoft Inc.
PackageName: RamSoft DicomProxy
License: Copyright © 1994-2021 RamSoft Inc. All rights reserved.
ShortDescription: DicomProxy is a windows appplication running as a windows service to assists local modalities to transfer dicom studies to a remote PACS server.
ManifestType: defaultLocale
ManifestVersion: 1.0.0
```

**Comment by ramsoft-inc** [comment]
@denelon @ItzLevvie
PR build failed on installation validation without any specific error message, can you please help me to fix the issue?

**Comment by ramsoft-inc** [comment]
@denelon Again same failure

**Comment by denelon** [comment]
```
2021- 6-17 13:42:53.068 [CLI ] Package hash verification failed. SHA256 in manifest [2fea46fae58a8ff1f45b143440fd832d5a00f182ed832be16d097f457f4350e3] does not match download [32d9f632c111bdfd7702528c90c3ee8377a8fee8c1cb65a6508fb2f6e21154c9]
```

It still looks like we're getting a hash error.

**Comment by ramsoft-inc** [comment]
@denelon  I already tried with that hash https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=181402&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744  than i get back to original https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=181379&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744.

And last time with my 2fea46fae58a8ff1f45b143440fd832d5a00f182ed832be16d097f457f4350e3 installer scan was passed but   Installation Validation was filed https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=181662&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=58bdf892-4e45-5ba7-4782-31912f95e167

**Comment by KevinLaMS** [comment]
My quick test shows that manifest includes hash 2fea46f*
But when I download the package. the hash reported by  winget hash is 32d9f632*.

**Comment by ramsoft-inc** [comment]
@KevinLaMS  https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=181402&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744

I tired that hash as well, it was failed.

**Comment by KevinLaMS** [comment]
@ramsoft-inc .  Finally I forced the package to install despite the hash error.  And I am not seeing that anything got installed.  This explains the why you were also getting the  Validation-Installation-Error .
What files is this installer putting down?  How can I verify the install is working correctly.  
In the meantime I will menace my dev to see if they can figure out what is going on with the HASH.

**Comment by ramsoft-inc** [comment]
> @ramsoft-inc . Finally I forced the package to install despite the hash error. And I am not seeing that anything got installed. This explains the why you were also getting the Validation-Installation-Error .
> What files is this installer putting down? How can I verify the install is working correctly.
> In the meantime I will menace my dev to see if they can figure out what is going on with the HASH.

@KevinLaMS
You may check this location C:\dicom-proxy-source\Ramsoft.DicomProxy. If you found files here, installation is good.

**Comment by ItzLevvie** [comment]
Newer package seems to be available at https://github.com/microsoft/winget-pkgs/tree/master/manifests/r/RamSoft/DicomProxy/0.2.259-ci.4

**Pull request closed without merging**